src/assets/scripts/client/aircraft/Pilot/Pilot.js

```diff
@@ -816,23 +834,6 @@ export default class Pilot {
         return [true, this._mcp.speed];
     }
 
-    /**
-     * Set the arrival runway to the specified runway model
-     *
-     * @for Pilot
-     * @method setArrivalRunway
-     * @param aircraft {AircraftModel}
-     * @param runwayModel {RunwayModel}
-     * @return {array} [success of operation, response]
-     */
-    setArrivalRunway(aircraft, runwayModel) {
-        if (aircraft.isOnGround()) {
-            return [false, 'unable to accept arrival runway assignment until airborne'];
-        }
```

Comment:
It seems that you've replaced `Pilot.setArrivalRunway()` with `Pilot.updateStarLegForArrivalRunway()`. Initially, that concerned me because it sounds like it would be an issue with routes that don't include STARs (which are supposed to be supported-- fix to fix to fix to the airport, etc). But as it turns out, `RouteModel.isRunwayModelValidForStar()` will be true if there is no STAR leg found, so everything should actually work out correctly. Just something to be aware of.